api/hyp3_api/auth.py

```diff
@@ -0,0 +1,29 @@
+import time
+from os import environ
+
+import jwt
+
+
+def decode_token(token, required_scopes):
+    try:
+        jwt.decode(token, environ['AUTH_PUBLIC_KEY'], algorithms=environ['AUTH_ALGORITHM'])
```

Comment:
`decode()` can deduce the algorithm for RSA256, dunno if `environ['AUTH_ALGORITHM']` should be optional or not, probably better to make it required

Reply:
It is going to be made required in a future version of PyJWT so is reccomended to be provided